pkg/server/plugin/datastore/sql/sql_test.go

```diff
@@ -1236,6 +1236,36 @@ func (s *PluginSuite) TestListRegistrationEntriesAgainstMultipleCriteria() {
 	s.RequireProtoListEqual([]*common.RegistrationEntry{entry}, resp.Entries)
 }
 
+func (s *PluginSuite) TestListRegistrationEntriesWhenCruftRowsExist() {
+	_, err := s.ds.CreateRegistrationEntry(ctx, &datastore.CreateRegistrationEntryRequest{
+		Entry: &common.RegistrationEntry{
+			Selectors: []*common.Selector{
+				{Type: "TYPE", Value: "VALUE"},
+			},
+			SpiffeId: "SpiffeId",
+			ParentId: "ParentId",
+			DnsNames: []string{
+				"abcd.efg",
+				"somehost",
+			},
+		},
+	})
+	s.Require().NoError(err)
+
+	// This is gross. Since the bug that left selectors around have been fixed,
```

Comment:
"the bug" please leave a link to the github issue here.

Reply:
Good call!  Added.